apps/android-service-runtime/src-tauri/src/lib.rs

```diff
@@ -13,7 +14,13 @@ pub fn run() {
     log::set_max_level(LevelFilter::Warn);
 
     tauri::Builder::default()
-        .plugin(tauri_plugin_holochain_service::init())
+        .plugin(tauri_plugin_holochain_service::init(
+            RuntimeNetworkConfigFfi {
+                bootstrap_url: "https://bootstrap-0.infra.holochain.org".to_string(),
+                signal_url: "wss://sbd.holo.host".to_string(),
+                ice_urls: vec!["stun:stun.l.google.com:19302".to_string()],
```

Comment:
Should we add cloudflare as well here? We've been using it in Moss as the primary one and it seems to work okay so far.

Reply:
added: https://github.com/holochain/android-service-runtime/pull/118/commits/0bb33172679e25996bd8429077bdf5eccc709b8f